src/roman_datamodels/maker_utils/_common_meta.py

```diff
@@ -141,6 +141,23 @@ def mk_outlier_detection(**kwargs):
     return od
 
 
+def mk_sky_background(**kwargs):
+    """
+    Create a dummy Sky Background instance with valid values for attributes
+    required by the schema. Utilized by the model maker utilities below.
+
+    Returns
+    -------
+    roman_datamodels.stnode.SkyBackground
+    """
+    sb = stnode.SkyBackground()
+    sb["level"] = kwargs.get("level", NONUM * (u.DN / u.s))
```

Comment:
Let's at least make the default units the MJy/sr units that make physical sense rather than DN / s.

Reply:
Changed.